PROVESFlightControllerReference/ReferenceDeployment/Top/ReferenceDeploymentTopology.cpp

```
@@ -118,6 +119,45 @@ void setupTopology(const TopologyState& state) {
+    // NOTE: because this runs after loadParameters(), any operator PRM_SAVE of CHANNEL_ENABLED will be
+    // silently overwritten by this default on the next boot -- that's intentional for now (safe default
+    // takes priority over a saved override) but worth revisiting if per-mission persistence is needed.
+    {
+        // BufferRepeater's paramSet_CHANNEL_ENABLED() is private (only the command-dispatch path may
+        // call it), so drive it the same way a real CHANNEL_ENABLED_PRM_SET command would: build the
+        // command argument buffer and invoke the component's cmdIn port directly. Opcode 0x0 is
+        // OPCODE_CHANNEL_ENABLED_SET (see generated BufferRepeaterComponentAc.hpp) -- the first/only
+        // settable param on this component, stable as long as BufferRepeater.fpp isn't changed.
+        Utilities::BufferRepeater_OutputChannelEnables uartOnlyChannelEnables(
+            {Fw::Enabled::ENABLED, Fw::Enabled::DISABLED, Fw::Enabled::DISABLED});
+        Fw::CmdArgBuffer channelEnablesArgs;
```

Clang (14.0.6)

[warning] variable 'channelEnablesArgs' is not initialized

(cppcoreguidelines-init-variables)